sdks/java/core/pom.xml

```diff
@@ -430,6 +430,19 @@
 
     <!-- test dependencies -->
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-common-fn-api</artifactId>
+      <type>test-jar</type>
```

Comment:
Yea that was a type / brain-o. I mean `classifier` vs `type`.

I had thought `classifier` was preferred because it does not specify a packaging type (an implementation detail) but rereading [the docs](https://maven.apache.org/pom.html), I'm pretty sure that leaving `type` unspecified just leaves you with the default of `jar`. In fact, some people seem to believe that `<classifier>tests</classifier>` is an abuse and using the `type` is better practice. I'm inclined to agree, though I'd say the whole design is a bit murky.

So my current thinking: good as-is and if/when we hit a technical reason to go one way or the other we should document the heck out of it.